frontend/src/components/orgFacilities/ManageOrganization.vue

```diff
@@ -0,0 +1,195 @@
+              <v-row no-gutters>
+                <v-col cols="12" sm="5" xl="4" xxl="3">
+                  <AppLabel>Province:</AppLabel>
+                </v-col>
+                <v-col cols="12" sm="7" xl="8" xxl="9">
+                  {{ organizationModel.province1 }}
+                </v-col>
+              </v-row>
+            </v-col>
+            <v-col cols="12">
+              <v-row no-gutters>
+                <v-col cols="12" sm="5" xl="4" xxl="3">
+                  <AppLabel>Postal Code:</AppLabel>
+                </v-col>
+                <v-col cols="12" sm="7" xl="8" xxl="9">
+                  {{ organizationModel.postalCode1 }}
+                </v-col>
+              </v-row>
+            </v-col>
+          </v-row>
+          <v-row no-gutters class="mt-6 mb-2">
+            <AppLabel>Physical Address</AppLabel>
+          </v-row>
+          <v-row no-gutters class="my-2">
+            <v-col cols="12" sm="5" xl="4" xxl="3">
+              <AppLabel>Street Address:</AppLabel>
+            </v-col>
+            <v-col cols="12" sm="7" xl="8">
+              {{ organizationModel.address2 }}
+            </v-col>
+            <v-col cols="12">
+              <v-row no-gutters class="my-2">
+                <v-col cols="12" sm="5" xl="4" xxl="3">
+                  <AppLabel>City:</AppLabel>
+                </v-col>
+                <v-col cols="12" sm="7" xl="8" xxl="9">
+                  {{ organizationModel.city2 }}
+                </v-col>
+              </v-row>
+            </v-col>
+            <v-col cols="12">
+              <v-row no-gutters>
+                <v-col cols="12" sm="5" xl="4" xxl="3">
+                  <AppLabel>Province:</AppLabel>
+                </v-col>
+                <v-col cols="12" sm="7" xl="8" xxl="9">
+                  {{ organizationModel.province2 }}
+                </v-col>
+              </v-row>
+            </v-col>
+            <v-col cols="12">
+              <v-row no-gutters class="my-2">
+                <v-col cols="12" sm="5" xl="4" xxl="3">
+                  <AppLabel>Postal Code:</AppLabel>
+                </v-col>
+                <v-col cols="12" sm="7" xxl="8">
+                  {{ organizationModel.postalCode2 }}
+                </v-col>
+              </v-row>
+            </v-col>
+          </v-row>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+<script>
+import { mapActions, mapState } from 'pinia';
+
+import { useOrganizationStore } from '@/store/ccof/organization.js';
+
+import AppLabel from '@/components/util/AppLabel.vue';
+
+export default {
+  name: 'ManageOrganization',
+  components: {
+    AppLabel,
+  },
+  data() {
+    return {
+      orgLoading: true,
+    };
+  },
+  computed: {
+    ...mapState(useOrganizationStore, ['organizationId', 'organizationModel']),
+  },
+  async mounted() {
+    await this.loadOrganization(this.organizationId);
+    this.orgLoading = false;
```

Comment:
The CCOF code base isn't too consistent about this but it's nice to throw any data loading in a try/catch otherwise you'll get stuck in a loading state if there's a failure. See frontend/src/components/eceweApplication/EceweEligibility.vue.

Reply:
Patched!